test-fixture/src/lib.rs

```diff
@@ -52,7 +52,7 @@ pub fn fixture_init() {
 
 // This needs to be > 2ms to avoid it being rounded to zero.
 // NSS operates in milliseconds and halves any value it is provided.
-pub const ANTI_REPLAY_WINDOW: Duration = Duration::from_millis(10);
+pub const ANTI_REPLAY_WINDOW: Duration = Duration::from_millis(1000);
```

Comment:
Why is this change needed?

Reply:
We had an earlier attempt to fix this that required a new test. That test exposed a bug in the current test fixtures that broke 0-RTT. This change fixes that issue.